README.md

```
@@ -1,143 +1,531 @@
+
+2. **Relationships**
+
+   ```typescript
+   // DO: Use type-safe references
+   authorId: uuid("author_id").references(() => profile.id, {
+     onDelete: "cascade",  // Specify behavior
+   }),
+   ```
+
+3. **Migrations**
+   ```typescript
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)